test/infrastructure/docker/templates/cluster-template-mp-development.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
You're probably just using this to test stuff. I would suggest to extend the existing clusterclass-quick-start.yaml to add MachinePools as then use the existing cluster-template

Would be good if you can make the changes here in the PR accordingly.

For your dev purposes you can move the files somewhere else and if you use tilt you can configure the folder where they are located